src/models/feed.rs

```diff
@@ -15,6 +15,7 @@ pub struct Feed {
     pub link: String,
     pub favicon_url: Option<String>,
     pub last_updated: DateTime<Utc>,
+    pub unread_count: i64,
```

Comment:
```suggestion
    pub unread_count: u16,
```

I would use a smaller unsigned int for this, such as `u16` for two reasons:
- Negative counts does not exist.
- I believe 16 bits is more than enough to count unread articles.